notebooks/official/training/tpuv5e_gemma_peft_finetuning_and_serving.ipynb

```diff
@@ -656,7 +608,7 @@
         "#### Kaggle credentials are required for KerasNLP training and Hex-LLM deployment with TPUs.\n",
         "Set the KAGGLE_USERNAME AND KAGGLE_KEY to pass in as an environment variable for Vertex Training to use\n",
         "Fenerate the Kaggle username and key by following [these instructions](https://github.com/Kaggle/kaggle-api?tab=readme-ov-file#api-credentials).\n",
-        "You will need to review and accept the model license as mentioned earlier."
+        "You need to review and accept the model license as mentioned earlier."
```

Comment:
suggested edit:

- As mentioned earlier, you need to review and accept the model license.

Reply:
made the change. Please review